content/software-review.md

```diff
@@ -0,0 +1,23 @@
++++
+title = "rOpenSci software review"
+description = "rOpenSci's open peer-review system of R packages contributed by the community"
++++
+
+rOpenSci's [suite of packages](/packages/) is partly contributed by staff members and partly contributed by community members, which means the suite stems from a great diversity of skills and experience of developers. How to ensure quality for the whole set? That's where our software system comes into play: packages contributed by the community undergo a **transparent, constructive, non adversarial and open review process**. For that process relying mostly on **volunteer work**, _[associate editors](/about#team)_ manage the incoming flow and ensure progress of submissions; _authors_ create, submit and improve their package; _[reviewers](https://github.com/ropensci/onboarding#reviewers)_, two per submission, examine the software code and user experience.
```

Comment:
> rOpenSci’s suite of packages is partly contributed by staff members and partly contributed by community members

rOpenSci’s suite of packages is comprised of contributions from staff engineers and the wider R community.

Reply:
> which means the suite stems from a great diversity of skills and experience of developers.

bringing considerable diversity of skills, expertise and experience to bear on the suite.

Reply:
>  How to ensure quality for the whole set?

We don't evaluate the whole set. We do it by package. So you can say: How do we ensure that every package is held to a high standard?